components/navbar/navbar.css

```diff
@@ -0,0 +1,52 @@
+/* @import url('../../components/root.css');
+@import url('../../components/list/list.css');
+@import url('../../components/toast/toast.css'); */
+
+@import url('../../components/framer.css');
+
+.navbar{
+    padding:1rem;
+    background: white;
+    color: rgb(22, 21, 21);
+}
+
+.navbar-primary{
+    background: var(--col-primary);
+}
+.navbar-danger{
+    background: var(--col-danger);
+}
+
+.navbar-dark{
+    background: var(--col-dark);
+}
+
+.navbar-white-text{
+    color: var(--text-col-white);
+}
+
+.navbar-dark-text{
+    color: var(--col-dark);
+}
+
+.navbar-icons{
+    padding: 0.5rem;
+    font-size: 1.3rem !important;
+    cursor: pointer;
+}
```

Comment:
Suggestion: To avoid `!important` tags.
Depends on the usecase, if you are trying to force the color then its fine 👍